assets/app/views/create.html

```diff
@@ -1,119 +1,117 @@
-<div ng-controller="ProjectController">
-  <div ng-controller="CreateController" class="add-to-project">
-    <div class="container">
-      <ol class="breadcrumb">
-        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
-        <li class="active"><strong>Add to Project</strong></li>
-      </ol>
-      <div class="row">
-        <div class="col-md-12">
-          <p ng-if="emptyCatalog && !loaded">Loading...</p>
+<div class="add-to-project">
+  <div class="container">
+    <ol class="breadcrumb">
+      <li><a href="{{project.metadata.name | projectOverviewURL}}">{{(project | displayName) || project.metadata.name}}</a></li>
```

Comment:
One example: `projectName` was a route param available immediately before. Now it won't update until the project loads.